### ISSUE TITLE
My account

### ISSUE THREAD

**Issue by Troshchuk** [top1r/testrepo#1]
Why do you use my account?

**Comment by top1r** [comment]
Hello mate.
The issue is that you have your account logged in on a bionic notebook.
I'm going to remove you soon :)

Apologize for the inconvenience.

Have a nice evening.

Regards,
Alexander

**Comment by Troshchuk** [comment]
Do you know my password?

**Comment by top1r** [comment]
No. It looks like you have your ssh key added to the global git config.
Does this makes sense for you?

**Comment by Troshchuk** [comment]
Do u have access now?

**Comment by top1r** [comment]
Don't know, pal. I am out of the school as of my last message.
But if you've removed your key from github, I think we are all set.
